app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCPane.scala

```diff
@@ -52,48 +44,66 @@ class DLCPane(glassPane: VBox) {
     onAction = new EventHandler[ActionEvent] {
       override def handle(event: ActionEvent): Unit = model.onOffer()
     }
+    tooltip = Tooltip(
+      "Initiates a DLC with the given oracle and contract parameters, generating an Offer message.")
+    tooltip.value.setShowDelay(new javafx.util.Duration(100))
   }
 
   private val acceptButton = new Button {
     text = "Accept"
     onAction = new EventHandler[ActionEvent] {
       override def handle(event: ActionEvent): Unit = model.onAccept()
     }
+    tooltip = Tooltip("In response to an Offer, generates an Accept message.")
+    tooltip.value.setShowDelay(new javafx.util.Duration(100))
   }
 
   private val signButton = new Button {
     text = "Sign"
     onAction = new EventHandler[ActionEvent] {
       override def handle(event: ActionEvent): Unit = model.onSign()
     }
+    tooltip = Tooltip("In response to an Accept, generates a Sign message.")
+    tooltip.value.setShowDelay(new javafx.util.Duration(100))
   }
 
   private val addSigsButton = new Button {
     text = "Add Sigs"
     onAction = new EventHandler[ActionEvent] {
       override def handle(event: ActionEvent): Unit = model.onAddSigs()
     }
+    tooltip = Tooltip("In response to a Sign, saves signatures.")
+    tooltip.value.setShowDelay(new javafx.util.Duration(100))
   }
 
   private val getFundingButton = new Button {
     text = "Get Funding Tx"
     onAction = new EventHandler[ActionEvent] {
       override def handle(event: ActionEvent): Unit = model.onGetFunding()
     }
+    tooltip = Tooltip(
+      "After adding signatures, generates and broadcasts the DLC to the blockchain.")
+    tooltip.value.setShowDelay(new javafx.util.Duration(100))
   }
 
   private val refundButton = new Button {
     text = "Refund"
     onAction = new EventHandler[ActionEvent] {
       override def handle(event: ActionEvent): Unit = model.onRefund()
     }
+    tooltip = Tooltip(
+      "After the refund timeout, broadcasts the refund transaction to the blockchain.")
```

Comment:
`Paste in contract id to broadcast the refund transaction`

Reply:
Again, I don't think the tooltip is the place to ask for an action like "paste" or to mention a contract id is needed, those are asked for after you click the button. The goal of the tooltip is to let the user know what the button will do and when to press it